pyrobosim/test/core/test_room.py

```diff
@@ -63,3 +64,26 @@ def test_add_room_to_world_in_collision(self, caplog):
         assert result is None
         assert world.num_rooms == 1
         assert "Room room1 in collision. Cannot add to world." in caplog.text
+
+    def test_add_room_with_different_color_formats(self):
```

Comment:
You could also consider turning this into a unit test for the `parse_color()` function itself under `pyrobosim/test/utils` so you don't have to recreate the same test for locations, hallways, objects, robots, etc. when those land.

Reply:
Nice suggestion !! and hence, created separate file: `test_general_utils.py` for this.